bom/application/pom.xml

```diff
@@ -4168,6 +4168,12 @@
                 <groupId>org.apache.httpcomponents</groupId>
                 <artifactId>httpclient-cache</artifactId>
                 <version>${httpclient.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>commons-logging</groupId>
+                        <artifactId>*</artifactId>
```

Comment:
So this change was apparently already partially applied (see the artifact just above) but with a wildcard so I kept the wildcard for this one too.